src/components/CityDetails/Modal/index.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+import "animate.css/animate.min.css";
+import "./styled.scss";
+
+function Modal({ content, onClose, isModalOpen }) {
+  console.log(isModalOpen);
+  return (
+    <div
+      className={`animate__animated ${
+        isModalOpen ? "animate__fadeIn" : "animate__fadeOut"
+      } modal`}
+    >
+      <div className="modal-content">
+        <span className="close" onClick={onClose}>
+          &times;
+        </span>
+        <p>{content}</p>
+      </div>
+    </div>
+  );
+}
+
+export default Modal;
+
+Modal.propTypes = {
+  content: PropTypes.node.isRequired,
+  onClose: PropTypes.func.isRequired,
+  isModalOpen: PropTypes.func.isRequired,
+};
```

Comment:
There is a rule that `export default` is the last line of a file.
```suggestion
Modal.propTypes = {
  content: PropTypes.node.isRequired,
  onClose: PropTypes.func.isRequired,
  isModalOpen: PropTypes.func.isRequired,
};

export default Modal;

```

Reply:
Fixed